rpc/types/src/macros.rs

```diff
@@ -16,11 +16,11 @@
 ///
 /// # Macro internals
 /// This macro uses:
-/// - [`__define_request`]
-/// - [`__define_response`]
-/// - [`__define_request_and_response_doc`]
+/// - [`define_request`]
+/// - [`define_response`]
+/// - [`define_request_and_response_doc`]
 ///
-/// # `__define_request`
+/// # `define_request`
```

Comment:
With custom epee impls, we actually need to use some of these macros individually elsewhere, so no `__` prefix.